caps/ajio_local_ios_capabilities.json

```diff
@@ -0,0 +1,27 @@
+{
+  "iOS": {
+    "platformName": "iOS",
+    "app": "src/test/resources/ajio.zip",
```

Comment:
you can point to https://github.com/anandbagmar/sampleAppsForNativeMobileAutomation/ - if the app is no there, raise a PR to add it there

Reply:
PR raised

Reply:
As discussed we will not be pushing it in repo as the app is internal to AJIO.